src/api/todos.ts

```diff
@@ -5,4 +5,6 @@ export const getTodos = (userId: number) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
-// Add more methods here
+export const postTodos = (userId: number, data: Todo) => {
```

Comment:
you create just 1 todo
```suggestion
export const createTodo = (userId: number, data: Todo) => {
```